src/libraries/System.Text.Json/src/System/Text/Json/Serialization/ReadStack.cs

```diff
@@ -411,6 +411,6 @@ private void SetConstructorArgumentState()
         }
 
         [DebuggerBrowsable(DebuggerBrowsableState.Never)]
-        private string DebuggerDisplay => $"Path:{JsonPath()} Current: ConverterStrategy.{Current.JsonTypeInfo?.PropertyInfoForTypeInfo.ConverterStrategy}, {Current.JsonTypeInfo?.Type.Name}";
+        private string DebuggerDisplay => $"Path:{JsonPath()} Current: ConverterStrategy.{Current.JsonTypeInfo?.Converter.ConverterStrategy}, {Current.JsonTypeInfo?.Type.Name}";
```

Comment:
Kind should be good enough here I think

Reply:
I'll leave that one since it specifically reports on converter strategy. Can change in the future if we find that it's less relevant.